homeassistant/components/telegram_bot/__init__.py

```diff
@@ -250,6 +253,19 @@
     }
 )
 
+SERVICE_SCHEMA_SET_MESSAGE_REACTION = vol.Schema(
+    {
+        vol.Optional(CONF_CONFIG_ENTRY_ID): cv.string,
+        vol.Required(ATTR_MESSAGEID): vol.Any(
+            cv.positive_int, vol.All(cv.string, "last")
+        ),
+        vol.Required(ATTR_CHAT_ID): vol.Coerce(int),
+        vol.Required(ATTR_REACTION): vol.All(cv.ensure_list, [cv.string]),
```

Comment:
the docs say that only 1 emoji is supported. if that is the case, why make this a list?

Reply:
Since it is user facing, I would prefer to have a single value to avoid users getting confused by the format (and also to have proper input validation)
We can always change it later if Telegram changes.

Reply:
@abmantis thanks for the input. done!